Calendar/BookingHelpers/BookingGenericFunctions.ps1

```diff
@@ -0,0 +1,132 @@
+﻿function SplitDomainFromEmail {
+    param([string] $email)
+    return [string]$email.Split("@")[1]
+}
+
+function IsConsumerMailbox {
```

Comment:
>IsConsumerMailbox [](http://example.com/codeflow?start=9&length=17)

Some of these seem useful / generic.
think if you want to push them up to Calendar\Shared or \Shared.

Reply:
later stage